ophyd/areadetector/filestore_mixins.py

```diff
@@ -59,22 +60,102 @@ def _ensure_trailing_slash(path):
 class FileStoreBase(BlueskyInterface, GenerateDatumInterface):
     """Base class for FileStore mixin classes
 
-    The `write_path_template` and `read_path_template` are formatted using
-    `datetime.strftime`, which accepts the standard tokens, such as
-    %Y-%m-%d.
+    This class provides
+
+      - python side path management (root, seperate write / readback paths)
```

Comment:
I think 'readback' isn't right here, and this should say 'read path'. The readback value is actually the write_file_path.